app/controllers/entities_controller.rb

```diff
@@ -464,11 +480,24 @@ def new_entity_params
     params.require(:entity).permit(:name, :blurb, :primary_ext)
   end
 
+  def create_bulk_payload
+    payload = params.require('data').map do |record|
+      record.permit('type'       => 'entity',
```

Comment:
+1

Reply:
on second thought, i'm looking at this more closely and think it might fail b/c it doesn't permit `type => entity` (the other k/v pair in the jsonapi.org recommended schema). will run some tests real quick to see, but if i don't commit this refactor, that's why.

Reply:
actually: works just fine. it *will* allow incorrectly (or just wierdly) typed payloads like:

```json
data: [
  { 
    type: 'LOCKNESS_MONSTER',
    attributes: {
      name: 'nice monster',
      primary_ext: 'person',
      blurb: 'hmm...'
  }
]
```
which is okay because it will just throw away the "type" information. 

but sort of points up that the "type" information is sort of superfluous?

---

cc @aepyornis